eng/Version.Details.xml

```diff
@@ -53,7 +53,7 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>5535e31a712343a63f5d7d796cd874e563e5ac14</Sha>
     </Dependency>
-    <Dependency Name="System.Text.Json" Version="8.0.0">
+    <Dependency Name="System.Text.Json" Version="8.0.3">
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>5535e31a712343a63f5d7d796cd874e563e5ac14</Sha>
```

Comment:
This needs updating too. From https://nuget.info/packages/System.Text.Json/8.0.3 it looks like it should be

```suggestion
      <Sha>9f4b1f5d664afdfc80e1508ab7ed099dff210fbd</Sha>
```